### PR TITLE
Fix typo in TransportForceMergeAction and TransportClearIndicesCacheA…

### PR DESCRIPTION
**Pull request by mushao999**
This PR fix some typo in `TransportForceMergeAction` and `TransportClearIndicesCacheAction`

### CODE REVIEW

**Comment by idegtiarenko** [comment]
Jenkins test this

File: docs/changelog/88064.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pr: 88064
+summary: Fix typo in TransportForceMergeAction and TransportClearIndicesCacheAction
+area: Doc
+type: enhancement
+issues: []
```
Comment (idegtiarenko) [inline docs/changelog/88064.yaml:5]: Hello @mushao999, thanks for this javadocs corrections!
I believe javadocs are not considered to be a docs are. I have marked this change as `>non-issue` do you mind removing this yaml file to make the build pass?

Comment (mushao999) [reply docs/changelog/88064.yaml]: > Hello @mushao999, thanks for this javadocs corrections! I believe javadocs are not considered to be a docs are. I have marked this change as `>non-issue` do you mind removing this yaml file to make the build pass?

@idegtiarenko Thanks for you reply, yaml file has been removed.

**Comment by idegtiarenko** [comment]
Jenkins test this

**Review by idegtiarenko — APPROVED** [review]

**Comment by idegtiarenko** [comment]
Thank you!

**Pull request merged**